engines/account/app/services/account/signup_service.rb

```diff
@@ -23,7 +20,7 @@ def create!(email)
     # @return [Account::User]
     def complete!(signup, options)
       ::ActiveRecord::Base.transaction do
-        user = ::Account::User.create(
+        user = ::Account::User.create!(
           email: signup.email,
           username: options[:username],
           password: options[:password],
```

Comment:
The thing is that adfing the `!` makes transaction fail when validations fail. But we're not resque it in the controller en this is failing bad in the front-end.

What I did is this:

``` ruby
signup.destroy if user.persisted?
```